src/scopes.rs

```diff
@@ -30,8 +62,8 @@ pub enum BindingKind {
 
 #[derive(Clone, Copy, Debug, Eq, PartialEq, Hash)]
 pub enum ScopeKind {
-  Program,
-  Module,
+  // Module,
```

Comment:
Remove? I guess `Module` is just "root" scope that is handled implicitly?

Reply:
I removed it mainly because there was no usage related to it. I think adding it is not a bad option.